### ISSUE TITLE
expirationCacheDeleteExpired should be performed only 1 time per minute

### ISSUE THREAD

**Issue by sosnovsky** [FlowCrypt/flowcrypt-browser#5830]
`expirationCacheDeleteExpired` is fired each second, while it's expected to be performed only 1 time per minute.

To check it - uncomment [this line](https://github.com/FlowCrypt/flowcrypt-browser/blob/ac59d1f0f7e13aee66d37c2c2e704d91f96b3641/extension/js/common/browser/browser-msg.ts#L418) in `browser-msg.ts`

**Comment by ioanmo226** [comment]
It is because we are calling `messageRenderer.deleteExpired` in `gmail-element-replacer.everything` function which is called every seconds.
Do you want me to remove `messageRenderer.deleteExpired` call?

https://github.com/FlowCrypt/flowcrypt-browser/blob/748b9609d6625593e19b1b865c00e2b0fef85eeb/extension/js/content_scripts/webmail/gmail/gmail-element-replacer.ts#L155

**Comment by ioanmo226** [comment]
@sosnovsky

**Comment by sosnovsky** [comment]
Yes, I think there is no reason to check cache for expired records each second.
It'll be ok to run `deleteExpired` each minute, like we do in service worker.

Probably we can even do it directly in `MessageRenderer` - add interval function for removing expired records in the constructor of `MessageRenderer`. What do you think, can it work this way?

**Comment by ioanmo226** [comment]
Sounds good!
